modules/dedicated-managing-quotas-and-limit-ranges.adoc

```diff
@@ -8,7 +8,3 @@
 As an administrator, you are able to view, create, and modify quotas limit
```

Comment:
Missing an `and` between "quotas limit ranges".

Reply:
Fixed!